### DISCUSSION TITLE
Closed Channels Sorting

### DISCUSSION THREAD

**Discussion by NEEDcreations** [ZeusLN/zeus#1626] [Ideas]
It'd be great if we could sort the Closed Channels tab by "Date Closed." As it is, if one of my many channels closes and I don't notice it, it takes me awhile to figure out which one actually closed, if I can even figure it out. This would be a very convenient feature.

**Comment by kaloudis** [comment]
https://github.com/ZeusLN/zeus/pull/1672

Moving forward, please file feature requests as issues: https://github.com/ZeusLN/zeus/issues
